test/smoke/usm-globals/Makefile

```diff
@@ -10,9 +10,15 @@ CFLAGS       += -O3
 CLANG        = clang++
 OMP_BIN      = $(AOMP)/bin/$(CLANG)
 CC           = $(OMP_BIN) $(VERBOSE)
+
+SUPPORTED    = gfx90a
 #-ccc-print-phases
 #"-\#\#\#"
 
 include ../Makefile.rules
 run: $(TESTNAME)
+ifneq (,$(findstring $(INSTALLED_GPU),$(SUPPORTED)))
 	$(RUNENV) ./$(TESTNAME) 2>&1 | $(AOMP)/bin/FileCheck $(TESTSRC_MAIN)
+else
+	@echo "  Skipped running $(TESTNAME): test only supported on $(SUPPORTED)"
```

Comment:
Shouldn't this case be already covered by the upper Makefile rules output?
If we have that here, I wonder whether this may lead to doubly counting any skipped tests.

Reply:
So one is compiling the other is running.

Reply:
I can just print the message when we skip compilation and at this point leave only the THEN branch and forget about reporting the skipping of the running.